frontend/scss/app/components/_error_boundary.scss

```
@@ -0,0 +1,40 @@
+  gap: var(--padding-mini);
+  padding: var(--padding-default);
+  width: 100%;
+  box-sizing: border-box;
+  text-align: center;
+
+  &__icon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--color-error, #d9534f);
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)